hw_2.py

```diff
@@ -1,15 +1,105 @@
+import math
+import random
+import sys
+
+
 # 1) Write a function that emulates the game "rock, scissors, paper"
```

Comment:
General comment for Task1.

Try not to complicate the code. Here we no need a class at all, also we do not need so many objects like
self.tuple_combination_win, self.tuple_combination_draw, self.dict_variations_input and so much methods.  Will be enough one function.